pyproject.toml

```
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "Cython", "numpy"]
```

Add newline at end of file for git.
```suggestion
requires = ["setuptools", "wheel", "Cython", "numpy"]

```